### DISCUSSION TITLE
Check if a user is logged in through api

### DISCUSSION THREAD

**Discussion by lukewhitingdev** [mozilla/hubs#4587] [Hubs Cloud Questions] [answered]
Hi,

Wondering if there is a way to check if a user is currently logged in to a hubs cloud instance?, preferably remotely since I want to check login status on a web-portal before using hubs. 
Im aware of the api/v1 endpoints but cant seem to find anything in there to help me. 

Any help is appreciated

- Luke

**Comment by pattersonbl2** [comment, marked as answer]
We currently don't have a setup for this. We do have the discord bot integration that can do some user monitoring for you. if you want more information on this we have a page listed  [here](https://hubs.mozilla.com/docs/hubs-discord-bot.html).  I will open an issue behind this feature to possibly be worked on and added in the future .   https://github.com/mozilla/hubs/issues/4589
